packages/zipkin-instrumentation-kafkajs/src/kafka-recorder.js

```diff
@@ -12,10 +16,10 @@ const recordConsumeStart = (tracer, {topic, partition, message}) => {
 
     // TODO: this should definitely note join. It should make a child
     id = tracer.join(new TraceId({
-      traceId,
-      parentId: fromNullable(parentId),
-      spanId,
-      sampled: fromNullable(sampled),
+      traceId: bufferToAscii(traceId),
```

Comment:
we might consider moving this logic into traceId, if it is common for headers to be of type Buffer, though I think Buffer might be node.js only and so far this is the only user of it.